bedrock/settings/base.py

```diff
@@ -1907,6 +1914,67 @@ def before_send(event, hint):
     "RELAY_PRODUCT_URL", default="https://stage.fxprivaterelay.nonprod.cloudops.mozgcp.net/" if DEV else "https://relay.firefox.com/"
 )
 
+
+# Authentication with Mozilla OpenID Connect / Auth0 ============================================
+
+LOGIN_ERROR_URL = "/cms-admin/"
+LOGIN_REDIRECT_URL_FAILURE = "/cms-admin/"
+LOGIN_REDIRECT_URL = "/cms-admin/"
+LOGOUT_REDIRECT_URL = "/cms-admin/"
+
+OIDC_RP_SIGN_ALGO = "RS256"
+
+# How frequently do we check with the provider that the user still exists and is authorised?
+OIDC_RENEW_ID_TOKEN_EXPIRY_SECONDS = config(
+    "OIDC_RENEW_ID_TOKEN_EXPIRY_SECONDS",
+    default="64800",  # 18 hours
+    parser=int,
+)
+
+OIDC_CREATE_USER = False  # We don't want drive-by signups
```

Comment:
I've wondered about this. Do we not? If this were `True` then we'd be able to fully manage access via the group on people.m.o. I'm sure it makes this slightly more open, but it does sound like it'd make administration of users easier. Just curious about your thoughts.

Reply:
Hmm yeah - i get your take. TBH I think that was steered by the original advice I received for either Birdbox or actually mdn/developer-portal, which are designed to have very small user lists 

However, with Bedrock it's a different matter.

Happy to relax it given - as you say - we still have control via the peoplemo group.

Nice 👍

Reply:
Oh, this is it: when I set `OIDC_CREATE_USER` to `True`, the user is (understandably) made with `is_staff=False` and `is_superuser=False`, which denies them admin access.

We could add a `pre_save` Signal handler to set this appropriately for users with @mozilla.com email addresses, but I'll push this out to a separate Issue for now as it's a nice (definitely) to have

Reply:
Update: writing this up as an issue made me think about another angle.

We don't have any info flowing from people.m.o to say what kind of Pages/Snippets a user should have access to, so auto-creating them would either mean creating an `is_staff=True` user who can't do anything or (in the worst timeline) `is_superuser=True` who can do _everything_. Either way, we'd still need to manually tune their reach/role etc.

As such, I think sticking with `OIDC_CREATE_USER=False` is the better option because they'll need some manual setup. Maybe we can add something to bootstrap users more efficiently, but drive-by creation isn't enough on its own, alas.